test/src/extern/CMessageExtTest.cpp

```diff
@@ -41,8 +41,11 @@ TEST(cmessageExt, info) {
   mqMessageExt->setKeys("testKeys");
   EXPECT_EQ(GetMessageKeys(messageExt), mqMessageExt->getKeys());
 
-  mqMessageExt->setBody("testBody");
+  std::string body("testBody");
+  body.append(3, '\0');
+  mqMessageExt->setBody(body.c_str(), body.length());
   EXPECT_EQ(GetMessageBody(messageExt), mqMessageExt->getBody());
```

Comment:
This case is ok?

Reply:
The message body needs to support arbitrary binary data.